test/e2e_node/node_problem_detector.go

```diff
@@ -50,6 +50,8 @@ var _ = framework.KubeDescribe("NodeProblemDetector", func() {
 	var uid string
 	var ns, name, configName, eventNamespace string
 	var bootTime, nodeTime time.Time
+	initTime := time.Now()
```

Comment:
I wanted this called as early as possible, and not in the before each statement block.